scripts/binary/trigger-publish-binary-pipeline.ts

```diff
@@ -0,0 +1,33 @@
+import { exec } from 'child_process'
```

Comment:
Looks like we've got a few http libs available

```
$ ls node_modules | grep fetch
cross-fetch
isomorphic-fetch
make-fetch-happen
minipass-fetch
node-fetch
npm-registry-fetch
sync-fetch
whatwg-fetch
```

Should we just use one of those? It would make this quite a bit cleaner - you could just do `JSON.stringify()` etc.